### PR TITLE
Write code for TC05

### PR DESCRIPTION
**Pull request by lqthai07t3**
- Add method to try log in with several times in LoginPage.
- Add code for verify TC05 in LoginTest.

### CODE REVIEW

File: src/test/java/tests/LoginTests.java [hunk 1]
```diff
@@ -52,4 +53,21 @@ public void tc003_LoginWithInvalidPassword() {
...
+    @Test(testName = "System shows message when user enters wrong password several times")
+    public void tc005_LoginSeveralTimesWithInvalidPassword() {
+        LogHelper.info("Go to Login Tab");
+        homePage.clickLoginTab();
+
+        LogHelper.info("Log in with invalid password at the first time");
+        loginPage.login(Constants.USERNAME, Constants.PASSWORD + "142");
+
+        LogHelper.info("Log in with invalid password for 3 times");
+        loginPage.repeatLogin(DataHelper.getRandomPassword(12), 3);
```
Comment (MinhQuan119) [inline src/test/java/tests/LoginTests.java:66]: You can delete first time step and increase number of times on this step by 1

Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java]: It has been fixed, please check again.

**Review by MinhQuan119 — APPROVED** [review]

File: src/test/java/tests/LoginTests.java [hunk 2]
```diff
@@ -52,4 +53,18 @@ public void tc003_LoginWithInvalidPassword() {
...
                 "Error message is not displayed as expected");
     }
+
+    @Test(testName = "System shows message when user enters wrong password several times")
+    public void tc005_LoginSeveralTimesWithInvalidPassword() {
+        LogHelper.info("Go to Login Tab");
+        homePage.clickLoginTab();
+
+        LogHelper.info("Log in with invalid password for 4 times");
+        loginPage.repeatLogin(Constants.USERNAME, DataHelper.getRandomPassword(12), 4);
```
Comment (tanmle) [inline src/test/java/tests/LoginTests.java:63]: you dont need to random password

Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java]: Please check again. Random password has been removed in TC05

File: src/test/java/tests/LoginTests.java [hunk 3]
```diff
@@ -52,4 +53,18 @@ public void tc003_LoginWithInvalidPassword() {
...
                 "Error message is not displayed as expected");
     }
+
+    @Test(testName = "System shows message when user enters wrong password several times")
+    public void tc005_LoginSeveralTimesWithInvalidPassword() {
+        LogHelper.info("Go to Login Tab");
+        homePage.clickLoginTab();
+
+        LogHelper.info("Log in with invalid password for 4 times");
+        loginPage.repeatLogin(Constants.USERNAME, Constants.PASSWORD + "369", 4);
```
Comment (tanmle) [inline src/test/java/tests/LoginTests.java:63]: using `Constants.PASSWORD + "369"` is not wrong,, but you can just use `invalidPassword` or just a dummy string `qwertadsf`

Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java:63]: Okie, I think I will use invalidPassword for this case. Thank you :)

Comment (lqthai07t3) [reply src/test/java/tests/LoginTests.java]: I have fixed it. Please, check again.

File: src/main/java/common/Constants.java [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
-
-
-
-}
+    public static final String USERNAME = "rte@gmail.com";
+    public static final String PASSWORD = "36925814";
+    public static final String PID = " 12345678";
+
+    //INVALID ACCOUNT
+    public static final String INVALID_USERNAME = "rtedkd@gmail.com";
```
Comment (tanmle) [inline src/main/java/common/Constants.java]: no, this is not correct, you won't need to define the variables for invalid stuff in Constants

**Review by tanmle — APPROVED** [review]

File: src/main/java/common/Constants.java (hunk 4 above)
Comment (lqthai07t3) [reply src/main/java/common/Constants.java]: It has been fixed. Please check again :)

**Pull request merged**